rpcs3/rpcs3qt/update_manager.cpp

```diff
@@ -307,7 +307,9 @@ void update_manager::update(bool auto_accept)
 	{
 		if (m_update_message.isEmpty())
 		{
+			// This can happen if we abort the check_for_updates download. Just check again in this case.
 			m_downloader->close_progress_dialog();
+			check_for_updates(false, false, false, m_parent);
```

Comment:
As far as I can see the only way update_manager::update is called is at the end of update_manager::handle_json which means m_update_message is never empty? I don't see the path by which this can happen.

Reply:
It can be called from main_window

Reply:
Missed that, :+1: .